### DISCUSSION TITLE
cn/2024-09-20-/

### DISCUSSION THREAD

**Discussion by giscus** [JinjiPang/jinji#57] [Comments]
# cn/2024-09-20-/

一个月前的周日，我去罗德岛的一个海边冲浪。因为对自己在海里游泳的能力没有一个很好的认知，我游到了离海边已经有一段距离的地方去等浪，所处位置的海水深度远高于我的身高。海浪一波接一波，最初我在掉落冲浪板的时候还能在几秒内再爬回板子上，这让我变得盲目自信，在随后浪的高度和频率都增加了时候也没有及时往岸边游。随后，我被一道浪打下冲浪板，因为体力的消耗和浪频的增加，我数次尝试游到板子旁边都未果，急切游到板子 &hellip;

https://jinjipang.com/cn/2024-09-20-/

**Comment by XiangyunHuang** [comment]
死亡和迷茫都是不小的话题，也是挺能引起共鸣的话题，还是我们这个年龄段的话题。22年我好多时间都挺迷茫的，至今也没想清楚好多事，我的经验是别想了，有哪些自己感兴趣还没来得及做的事赶紧抽时间去做。

至于死亡，我少年时代有两次和你差不多的频临死亡的经验。我家乡在湖南，村前面有条河，夏天和同村一伙伴拿着一个大轮胎内胎去河里学游泳。想先把轮胎抛下去，然后跳进轮胎里，结果喝了好几口河水。

第二次的经历，和你更相似，我是去山塘里摸田螺。摸着摸着脚一滑，水就淹过脑袋，脚也够不着地了，浮力变大，感觉水里有股内流冲了我一下，我就滑到塘中去了。在我身体翻转过来之际，也是害怕极了，周围没有人，就我一个，垂死的扑腾，就像鸡一样，两只手使劲扑腾水，没想到，脚不知怎的触到地了（现在知道应该是手推水带来的反作用力，让我往岸边漂了）。然后，心有余悸地上了岸，从此，再没下过山塘，野泳，至今，也没学会游泳。

最后一个话题，人生的意义。我读过完全不同的话，陈丹青在某一期圆桌派说人生毫无意义（好像这期也谈死亡），他是对着官方主流意识形态和价值观去的，不要相信那些宣传赋予我们的意义。过自己才是真正的意义，我们能感受到、享受到的意义。

陈其钢在他的自传里，说人生有意义，并且驳斥那些说人生毫无意义的人是在追时髦话题。他其实也是在做自己、过自己，而且做的特别极致，力求作品与自己合二为一。在深山老林里，将音乐艺术当作自己生命的延续，做到极致便是他所追寻的意义。

尽管会受到周围环境和社会的影响，保留一块地方，还做自己就是意义。也许有一天，可以百分百身心地去做自己。

**Reply by XiangyunHuang** [reply]
我想硬 cue 益辉 @yihui 来谈谈，我感觉他比我们大10岁左右，也许过了我们这个年龄段，有一些蛮清晰的想法。

**Reply by yihui** [reply]
十天前读了这篇日志之后很震撼，确实很想写点什么，但尚在酝酿之中。等我把手头的几个小浪扑灭之后再来。

**Comment by qzh1880** [comment]
”越认识到我们生活的世界的客观存在，我们对死亡越恐惧。

死亡的恐怖源于意识、思想、记忆的消失。

我们对电子游戏中的死亡几乎没有恐惧，因为我们知道那只是一个程序，不会带走任何意识、思想、记忆。

从这种意义来说，梦的终止也是一种死亡，因为梦境的大部分记忆会随着清醒而离去，无法挽回。

可我们却并不惧怕做梦或醒来（顶多对遗忘梦境感到遗憾），这是因为我们知道，梦只是个梦，醒来迎接我们的是更精彩的现实生活。

于是，生命的终止，这种死亡，是绝对恐怖的。因为我们不能肯定，死亡如梦境的终止——还有一个更真实的世界等你。或者更糟糕的，如果我们选择相信现代科学，从神经元停止放电到细胞完全死亡，人在几分钟内失去意识、此生的全部思想和记忆，成为一堆没有思想分子。我们有一天会这样走向生命的终点，而物理学上的宇宙还在客观存在，这种悲哀和恐怖，已经难以用语言表达。

“哀吾生之须臾 羡长江之无穷”，苏轼大概与我共情，才会拿长江这个宇宙中客观的物，与自己的生命对比。“

这是我去年写的随笔，刚好也谈到这个话题，发来分享一下

关于人生意义和虚无，我上个月又写了一篇文章，在我的网站上
https://tallcat.cn/posts/meaning-of-life/

博主别把这个网站忘了，我在翻我以前收藏的独立博客，发现好多都停更了，挺可惜的

**Reply by XiangyunHuang** [reply]
人这一生，有多少时光是因为社会和国家的原因，又有多少时间是因为家庭的原因，还有多少是因为自己的原因，就那样毫无意义的过了。人这一生，要没有这些毫无意义的时光，都是有意义的时光，那这些有意义又从哪里来呢。无意义正添了这人生的厚度。说了这一段我自己都无法把握的话，见笑😜
